src/ui-kit/experimental/hierarchical/hierarchical-tree/hierarchical-tree.component.ts

```diff
@@ -51,6 +57,9 @@ export class SamHierarchicalTreeComponent implements OnInit {
    */
   public selectedValue: string;
 
+
+  private sort: Sort;
```

Comment:
@thomastighe Can you make sure you add two test cases to make sure sort works whether or not a Sort is provided? I don't see a change in your spec files in this PR.